### ISSUE TITLE
Build fails on fresh Arch install

### ISSUE THREAD

**Issue by suuuehgi** [ultrabug/py3status#1496]
```
==> Proceed with install? [Y/n] 
:: Parsing SRCINFO (1/1): py3status
==> Making package: py3status 3.12-1 (Mon 03 Sep 2018 05:51:19 PM CEST)
==> Retrieving sources...
  -> Found py3status-3.12.tar.gz
==> Validating source files with sha256sums...
    py3status-3.12.tar.gz ... Passed
==> Cleaning up...
==> Making package: py3status 3.12-1 (Mon 03 Sep 2018 05:51:19 PM CEST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found py3status-3.12.tar.gz
==> Validating source files with sha256sums...
    py3status-3.12.tar.gz ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Extracting py3status-3.12.tar.gz with bsdtar
==> Sources are ready.
==> Making package: py3status 3.12-1 (Mon 03 Sep 2018 05:51:20 PM CEST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Entering fakeroot environment...
==> Starting package()...
Traceback (most recent call last):
  File "setup.py", line 10, in <module>
    from setuptools.command.easy_install import _to_ascii, ScriptWriter
ImportError: cannot import name '_to_ascii' from 'setuptools.command.easy_install' (/usr/lib/python3.7/site-packages/setuptools/command/easy_install.py)
==> ERROR: A failure occurred in package().
    Aborting...
Error making: py3status
```

**Comment by suuuehgi** [comment]
Okay, this is already fixed but not updated on [AUR](https://github.com/ultrabug/py3status/pull/1413/files).

**Comment by lasers** [comment]
Fixed via #1508.
